src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.cpp

```diff
@@ -654,6 +654,23 @@ namespace UiaOperationAbstraction
         return m_useRemoteApi;
     }
 
+    UiaCacheRequest UiaOperationDelegator::CreateCacheRequest()
+    {
+        if (m_useRemoteApi)
+        {
+            return m_remoteOperation.NewCacheRequest();
+        }
+        else
+        {
+            winrt::com_ptr<IUIAutomation> automation;
+            THROW_IF_FAILED(::CoCreateInstance(__uuidof(CUIAutomation8), nullptr, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(automation.put())));
+

```

Comment:
You don't need it.

The abstraction relies on having one, pre-set instance of `IUIAutomation`:
```
wil::object_without_destructor_on_shutdown<wil::com_ptr<IUIAutomation>> g_automation;
```